File: CS/RazorSupport/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using RazorSupport.Models;
using DevExpress.Web.Mvc;

namespace RazorSupport.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Fill the form:";

            return TextBox();
        }
        public ActionResult TextBox() {
            return View("TextBox",
                new MyModel() { OrderDate = DateTime.Now, Name = "Test" });
        }

        [HttpPost]
        public ActionResult PostValues([ModelBinder(typeof(DevExpressEditorsBinder))] MyModel model) {
            ViewBag.Message = String.Format("Values from {0} were posted", model.Name);
            return View("TextBox", model);
        }
    }
}

File: VB/RazorSupport/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports RazorSupport.Models
Imports DevExpress.Web.Mvc

Namespace RazorSupport.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			ViewBag.Message = "Fill the form:"

			Return TextBox()
		End Function
		Public Function TextBox() As ActionResult
			Return View("TextBox", New MyModel() With {.OrderDate = DateTime.Now, .Name = "Test"})
		End Function

		<HttpPost> _
		Public Function PostValues(<ModelBinder(GetType(DevExpressEditorsBinder))> ByVal model As MyModel) As ActionResult
			ViewBag.Message = String.Format("Values from {0} were posted", model.Name)
			Return View("TextBox", model)
		End Function
	End Class
End Namespace
